nix/pkgs/xcbeautify/default.nix

```diff
@@ -5,14 +5,15 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "xcbeautify";
-  version = "1.1.1";
+  version = "1.4.0";
   arch = if stdenv.hostPlatform.isAarch64 then "arm64" else "x86_64";
 
+
   src = fetchurl {
-    url = "https://github.com/tuist/xcbeautify/releases/download/${version}/xcbeautify-${version}-${arch}-apple-macosx.zip";
+    url = "https://github.com/cpisciotta/xcbeautify/releases/download/${version}/xcbeautify-${version}-${arch}-apple-macosx.zip";
```

Comment:
Why did the repo owner change?

Reply:
As I see it was moved to a new owner. Previous link still works - it redirects.